Sources/BraintreePayPalMessaging/BTPayPalMessagingView.swift

```diff
@@ -76,17 +78,27 @@ public class BTPayPalMessagingView: UIView {
                     textAlign: request.textAlignment.textAlignmentRawValue
                 )
             )
-                        
-            let messageView = PayPalMessageView(config: messageConfig, stateDelegate: self, eventDelegate: self)
-            messageView.translatesAutoresizingMaskIntoConstraints = false
-            self.addSubview(messageView)
 
+            self.setupMessageView(withConfig: messageConfig)
+        }
+    }
+    
+    private func setupMessageView(withConfig config: PayPalMessageConfig) {
```

Comment:
Take it or leave it:
```suggestion
    private func setupMessageView(with config: PayPalMessageConfig) {
```